lenses/connect-customlens-globalResiliency.json

```diff
@@ -544,7 +569,7 @@
 				{
 					"id": "GlobalResiliency_09",
 					"title": "Are you checking whether agents have completed their active calls before failing them over?",
-					"description": "Shifting agent logins can be disruptive if they are actively participating in an interaction.",
+					"description": "Shifting agent logins can be disruptive if they are actively participating in a non-voice interaction.",

```

Comment:
Can you clarify this? The question implies it's about voice interactions, while the answers are talking about non-voice interactions.

Reply:
Updated to Interaction in the new commit, with Global Sign-in the agent will not be moved until the Voice call ends, that does not apply for non-voice